pkg/operator/cluster/cluster.go

```diff
@@ -103,15 +100,15 @@ func (c *Cluster) CreateInstance() error {
 		return fmt.Errorf("failed to create client access. %+v", err)
 	}
 
-	logger.Infof("Done creating rook instance in namespace %s", c.Spec.Namespace)
+	logger.Infof("Done creating rook instance in namespace %s", c.Namespace)
 	return nil
 }
 
 func (c *Cluster) Monitor(stopCh <-chan struct{}) {
 	for {
 		select {
 		case <-stopCh:
-			logger.Infof("Stopping monitoring of cluster %s", c.Spec.Namespace)
+			logger.Infof("Stopping monitoring of cluster %s", c.Namespace)
```

Comment:
should this be `c.Name` instead? (also, should both namespace and cluster name be logged?)